playbooks/gce/openshift-cluster/config.yml

```diff
@@ -15,10 +15,10 @@
 
 - include: ../../common/openshift-cluster/config.yml
   vars:
-    g_etcd_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-etcd' }}"
-    g_lb_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-lb' }}"
-    g_masters_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-master' }}"
-    g_nodes_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-node' }}"
+    g_etcd_group: "{{ groups['tag_host-type-openshift-etcd'] | intersect(groups['tag_environment_' ~ cluster_id}}"
```

Comment:
I just checked and GCE doesn't have tag_environment_, it has tag_env-.

For example: `tag_env-ops`

So, I think this should be:
```
    g_etcd_group: "{{ groups['tag_host-type-openshift-etcd'] | intersect(groups['tag_env-' ~ cluster_id }}"
```

Obviously we should fix this in the followup clusterid PR, but for this one, I think it should remain with what's there now (so it works).